web.sh

```diff
@@ -2,11 +2,11 @@
 
 # start web frontend
 
-. django_example/bin/activate
+source django_demo_venv/bin/activate
 
 if [ "${?}" -ne "0" ]; then
   echo "Error activating virtualenv: can't start dev server"
   exit 1
 fi
 
-python manage.py runserver --insecure
+make deploy
```

Comment:
is it possible to get rid of web.sh and setup.sh and have all this logic live in the Makefile? (too many setup scripts imo).

i.e. make setup should do the setup. and make deploy should run the setup and deploy the app.

i.e. git clone and make deploy is all you need to get everything installed and running